docs/basic/tests.md

```diff
@@ -96,7 +96,7 @@ contract_for_index_of = deal.chain(
     ),
     # element at this position is the first match
     deal.ensure(
-        lambda items, item, result: not any(el == item for el in items[:result]),
+        lambda items, item, result: item not in items[:result]),
```

Comment:
Tests fail:

https://github.com/life4/deal/actions/runs/11729658216/job/32697214453?pr=136

Could you remove the closing parenthesis, please?